docs/reference/glossary.md

```diff
@@ -32,6 +32,16 @@ An image that has no parent is a **base image**.
 [boot2docker](http://boot2docker.io/) is a lightweight Linux distribution made
 specifically to run Docker containers. The boot2docker management tool for Mac and Windows was deprecated and replaced by [`docker-machine`](#machine) which you can install with the Docker Toolbox.
 
+## bridge network
+
+A bridge network is a Link Layer device (also called a Layer 2 device) which
```

Comment:
maybe this should be just "A bridge"; the bridge _network_ uses a bridge to allow containers on the docker network to commjnicate with devices on the external network.

/cc @aboch